cloudformation/memsub-promotions-cf.json

```diff
@@ -100,14 +110,123 @@
       }
     }
   },
-
   "Resources": {
+    "MembershipSubPromotionsLambdaRole": {
+      "Type": "AWS::IAM::Role",
+      "Properties": {
+        "AssumeRolePolicyDocument": {
+          "Statement": [ {
+            "Effect": "Allow",
+            "Principal": {
+              "Service": [ "lambda.amazonaws.com" ]
+            },
+            "Action": [ "sts:AssumeRole" ]
+          } ]
+        },
+        "Path": "/",
+        "Policies": [ {
+          "PolicyName": "MembershipSub-Promotions-to-PromoCode-View-Lambda-Policy",
+          "PolicyDocument": {
+            "Version": "2012-10-17",
+            "Statement": [
+              {
+                "Effect": "Allow",
+                "Action": [
+                  "dynamodb:GetRecords",
```

Comment:
Maybe put these DynamoDB stream related permissions in a separate policy statement below?